### PR TITLE
Doc changes for T1LR configuration in AviInfraSetting

### PR DESCRIPTION
**Pull request by abhishekbsingh**

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: docs/crds/avinfrasetting.md [hunk 1]
```diff
@@ -99,6 +101,13 @@ AviInrfaSetting integrates with Openshift Routes, via the annotation
     aviinfrasetting.ako.vmware.com/name: "my-infrasetting"
 ```
 
+### Using Namespace Annotation
+AviInfraSetting also integrates with all the obove objects - Services/Ingresses/Routes, via the Namespace annotation.
```
Comment (akshayhavile) [inline docs/crds/avinfrasetting.md:105]: nit: `above`

File: docs/crds/avinfrasetting.md [hunk 2]
```diff
@@ -99,6 +101,13 @@ AviInrfaSetting integrates with Openshift Routes, via the annotation
...
+
+```
+  annotations:
+    aviinfrasetting.ako.vmware.com/name: "my-infrasetting"
+```
```
Comment (akshayhavile) [inline docs/crds/avinfrasetting.md:110]: can we please elaborate more on this? Like will aviinfrasetting be applied to all ingress, svc objects present inside that namespace? What if ingress class and namespace both have aviinfrasetting? Which will take precedence?

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: docs/crds/avinfrasetting.md (hunk 2 above)
Comment (abhishekbsingh) [reply docs/crds/avinfrasetting.md:110]: done, thanks.

**Comment by abhishekbsingh** [comment]
skip-jira-id-check

**Comment by abhishekbsingh** [comment]
build ako

**Comment by abhishekbsingh** [comment]
build ako

**Comment by abhishekbsingh** [comment]
skip jira-id-check

**Comment by abhishekbsingh** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Review by apalsule — APPROVED** [review]

**Pull request merged**